packages/statemanager/test/ethersStateManager.spec.ts

```diff
@@ -182,7 +182,7 @@ tape('runTx custom transaction test', async (t) => {
     // The `MockProvider` is not able to load JSON files dynamically in browser so skipped in browser tests
     t.end()
   } else {
-    const common = new Common({ chain: Chain.Mainnet })
+    const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.London })
```

Comment:
Why do we set this to London?

Reply:
Test was failing with the latest HF updates to Shanghai default, so I set the to something fixed (if I recall correctly it was something test setup related).